code/datums/uplink/medical.dm

```diff
@@ -53,3 +53,9 @@
 	item_cost = 36 // Lowered, for traitors.
 	antag_costs = list(MODE_MERCENARY = 48)
 	path = /obj/item/storage/firstaid/combat
+
+/datum/uplink_item/item/medical/zombiebottle
+	name = "Unknown Serum"
+	desc = "A bottle containing some sort of unknown crimson red serum, I hope you know what you are doing with it."
+	item_cost = 150 // Collaborate, or we have endless zombie rounds
```

Comment:
Again, I strongly object to this balancing. 

For this price, you could buy 7 para-pens or 18 frag grenades and murderbone half the station, so I don't buy the argument that this is an effective way to make things fun - since less fun solutions are much cheaper.

Second, even if you herd cats long enough to get some taters to pool resources, all this buys you _one_ dose. Less than one, if you keep it at the 5u bottle. And the zombie serum can be hard-countered by medical.

So... you went through the pain of aligning fellow antags on the plan, bought the dose... and now you're screwed, because you don't even have a good set of resources to _apply_ it to a nonantag, because you spent them all on the reagent alone.

Okay, so how about a traitor self-zombifies? Well, by the balance of odds, they're as good as dead, because zombies are S L O W and our regulars have figured out anti-zombie measures.

If you want to gate zombie rounds behind some effort, what I'd suggest instead is adding a new chemical pathway to create zombie serum and giving antags a _precursor_. 

This would be substantially more fun, because a traitor who wants to trigger zombies now implicitly gets secondary objectives required to do that. 

Let's say the reaction is, arbitrarily, `z_precursor + devilsblood + dylovene @ 100K` - now the antag needs to either break into the bar or persuade the bartender for the drink, buy or steal some medicine, and find a chemical cooler to synthesize it.

Antag teams can collaborate to procure different steps of the puzzle, and at the end they can still kidnap nonantags to zombify and have tools to do so. And even if they don't manage it in the end, the crew will have to deal with a criminal conspiracy stealing seemingly random crap, or be corrupted into assisting them for $$$ (since what they ask for seems innocuous enough).

Reply:
Resolved w/ TC Cost Reduction after discussion